src/ArborX_Config.hpp.in

```diff
@@ -0,0 +1,17 @@
+/****************************************************************************
+ * Copyright (c) 2012-2019 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_CONFIG_HPP
+#define ARBORX_CONFIG_HPP
+
+#cmakedefine ArborX_ENABLE_MPI
```

Comment:
Use all capital for defines

Reply:
Is it fine with you to use `ARBORX_ENABLE_MPI` as `CMake` option as well? Otherwise, we can't use `#cmakedefine`.

Reply:
I vote for merge as is and fix later if necessary

Reply:
I would prefer `ArborX_ENABLE_MPI` as CMake option, and `ARBORX_ENABLE_MPI` as macro.